.github/workflows/main.yml

```diff
@@ -0,0 +1,94 @@
+name: Godot CI
+
+on:
+    pull_request:
+    push:
+      branches: [ master, 3.2, 3.1, 3.0 ]
+env:
+  SCONS_CACHE_MSVC_CONFIG: true
+  SCONS_CACHE_LIMIT: 2048
+jobs:
+  windows-editor:
+    runs-on: "windows-latest"
+    name: Windows Editor (release_debug)
+    steps:
+    - uses: actions/checkout@v2 
+    # cache must be put in the github workspace directory on windows to be persisted correctly
+    - name: "force cache path to be correct"
```

Comment:
Some steps start with capital letters, others with lowercase. We should be consistent (probably capital letter everywhere).

Reply:
I have fixed this in the code it should now be consistent